### ISSUE TITLE
Bot not responding

### ISSUE THREAD

**Issue by AlanMRichards** [OfficeDev/microsoft-teams-apps-faqplus#80]
Hi,

I have installed everything using the guide but the bot won't respond to any user input in either the web chat test in Azure or if I deploy to Teams. The QnA test option responds fine.

**Comment by kozziag** [comment]
> Hi,
> 
> I have installed everything using the guide but the bot won't respond to any user input in either the web chat test in Azure or if I deploy to Teams. The QnA test option responds fine.

I am experiencing the same issue. I am wondering if the deployment is missing some files. For testing I downloaded latest rep zip and loaded it into my own repo instead of forking or closing existing repo. Then edited the Deployment json and ran it. It actually added more items. Example function app component. The deployment failed loading 3 components with an error but at least I was able to see more items. I think the Template updated in Dec 2020 may have an issue and needs to be revalidated. 

Has anyone successfully used this template and have a working POC on or after Jan 1st, 2021?
